net-mail/swaks/swaks-20190914.0.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Swiss Army Knife SMTP; Command line SMTP testing, including TLS and AUTH"
+HOMEPAGE="http://www.jetmore.org/john/code/swaks"
```

Comment:
Use https https://jetmore.org/john/code/swaks/

Reply:
Fixed and rebased